sample-ovn-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "operator.openshift.io/v1"
+kind: "Network"
+metadata:
+  name: "cluster"
+spec:
+  serviceNetwork:
+    - "172.30.0.0/16"
+  clusterNetwork:
+    - cidr: "10.128.0.0/14"
+      hostPrefix: 23
+  defaultNetwork:
+    type: OVNKubernetes
+    RovnKubernetesConfig:
```

Comment:
typo. also, we still don't need this file

Reply:
Fixed typo. It was not obvious how to code this.